src/GroupTypeManagerInterface.php

```diff
@@ -102,7 +102,7 @@ public function getAllGroupContentBundlesByEntityType($entity_type_id);
    *   The bundle ID of the group content type for which to return associated
    *   group bundle IDs.
    *
-   * @return array
+   * @return string[]
```

Comment:
This is a nested array which is keyed by entity type ID, so this is a `string[][]`.